luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -112,10 +125,30 @@ public KeyValueStore getKeyValueStore(String keyValueStoreName) throws Unsupport
         }
     }
 
+    /**
+     * Retrieve or build a PhysicalTable.
+     *
+     * @param tableName the name for the PhysicalTable to be retrieved or built.
+     *
+     * @return the dimension instance corresponding to this name.
+     */
+    public ConfigPhysicalTable getPhysicalTable(String tableName) {
+        PhysicalTableDictionary physicalTableDictionary = resourceDictionaries.getPhysicalDictionary();
+        if (! physicalTableDictionary.containsKey(tableName)) {
+            ConfigPhysicalTable physicalTable = physicalTableFactoryPark.buildEntity(tableName, this);
+            physicalTableDictionary.put(tableName, physicalTable);
+        }
+        return physicalTableDictionary.get(tableName);
+    }
+
+    public DataSourceMetadataService getMetadataService() {
+        return metadataService;
+    }
 
     @Override
     public void load() {
         dimensionFactoryPark.fetchConfig().fieldNames().forEachRemaining(this::getDimension);
```

Comment:
I don't think we need this line anymore. Dimensions should be configured on demand as physical
table suse them. That being said, we can probably remove this in a final pass so that you don't
have to worry about tests exploding or something.

Reply:
Sure. 
The dimensions are not configured on-demand as for now, though (it is now being configured when the LIP is loaded). We can change that later, as I do not see that change can drastically change things.